### PR TITLE
Make services only log to stdout

### PR DESCRIPTION
**Pull request by fdalmaup**
|Related issue|
|---|
| #27098 |

## Description

Removes the file loggers for the Management API, Comms API, and Cluster and makes the services only log to `stdout` avoiding race conditions from multiple processes logging to the same file.
Manual testing can be found in the [issue](https://github.com/wazuh/wazuh/issues/27098#issuecomment-2514447656).

## Unit Tests

<details><summary>Related unit tests</summary>
```
====================================================================== test session starts ======================================================================
platform linux -- Python 3.10.15, pytest-7.3.1, pluggy-1.5.0
rootdir: /home/fdalmau/git/wazuh/api/api
configfile: pytest.ini
plugins: anyio-4.1.0, metadata-3.1.1, trio-0.8.0, tavern-1.23.5, cov-4.1.0, asyncio-0.18.1, html-2.1.1
asyncio: mode=auto
collected 6 items                                                                                                                                               

api/api/test/test_alogging.py ......                                                                                                                      [100%]

======================================================================= warnings summary ========================================================================
../../venv/5.0-aca/lib/python3.10/site-packages/connexion/json_schema.py:16
../../venv/5.0-aca/lib/python3.10/site-packages/connexion/json_schema.py:16
  /home/fdalmau/venv/5.0-aca/lib/python3.10/site-packages/connexion/json_schema.py:16: DeprecationWarning: jsonschema.RefResolver is deprecated as of v4.18.0, in favor of the https://github.com/python-jsonschema/referencing library, which provides more compliant referencing behavior as well as more flexible APIs for customization. A future release will remove RefResolver. Please file a feature request (on referencing) if you are missing an API for the kind of customization you need.
    from jsonschema import Draft4Validator, RefResolver

../../venv/5.0-aca/lib/python3.10/site-packages/connexion/json_schema.py:17
  /home/fdalmau/venv/5.0-aca/lib/python3.10/site-packages/connexion/json_schema.py:17: DeprecationWarning: jsonschema.exceptions.RefResolutionError is deprecated as of version 4.18.0. If you wish to catch potential reference resolution errors, directly catch referencing.exceptions.Unresolvable.
    from jsonschema.exceptions import RefResolutionError, ValidationError  # noqa

../../venv/5.0-aca/lib/python3.10/site-packages/connexion/validators/form_data.py:4
../../venv/5.0-aca/lib/python3.10/site-packages/connexion/validators/form_data.py:4
  /home/fdalmau/venv/5.0-aca/lib/python3.10/site-packages/connexion/validators/form_data.py:4: DeprecationWarning: Accessing jsonschema.draft4_format_checker is deprecated and will be removed in a future release. Instead, use the FORMAT_CHECKER attribute on the corresponding Validator.
    from jsonschema import ValidationError, draft4_format_checker

../../venv/5.0-aca/lib/python3.10/site-packages/connexion/validators/json.py:6
../../venv/5.0-aca/lib/python3.10/site-packages/connexion/validators/json.py:6
  /home/fdalmau/venv/5.0-aca/lib/python3.10/site-packages/connexion/validators/json.py:6: DeprecationWarning: Accessing jsonschema.draft4_format_checker is deprecated and will be removed in a future release. Instead, use the FORMAT_CHECKER attribute on the corresponding Validator.
    from jsonschema import Draft4Validator, ValidationError, draft4_format_checker

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================================================= 6 passed, 7 warnings in 0.36s =================================================================
====================================================================== test session starts ======================================================================
platform linux -- Python 3.10.15, pytest-7.3.1, pluggy-1.5.0
rootdir: /home/fdalmau/git/wazuh/framework
configfile: pytest.ini
plugins: anyio-4.1.0, metadata-3.1.1, trio-0.8.0, tavern-1.23.5, cov-4.1.0, asyncio-0.18.1, html-2.1.1
asyncio: mode=auto
collected 164 items                                                                                                                                             

framework/wazuh/core/config/models/tests/test_central_config.py .                                                                                         [  0%]
framework/wazuh/core/config/models/tests/test_comms_api.py ..............                                                                                 [  9%]
framework/wazuh/core/config/models/tests/test_engine.py ..........                                                                                        [ 15%]
framework/wazuh/core/config/models/tests/test_indexer.py .......                                                                                          [ 19%]
framework/wazuh/core/config/models/tests/test_logging.py .................                                                                                [ 29%]
framework/wazuh/core/config/models/tests/test_management_api.py ..........................                                                                [ 45%]
framework/wazuh/core/config/models/tests/test_server.py ..................................................................                                [ 85%]
framework/wazuh/core/config/models/tests/test_ssl_config.py ...............                                                                               [ 95%]
framework/wazuh/core/config/tests/test_client.py ........                                                                                                 [100%]

====================================================================== 164 passed in 0.32s ======================================================================
====================================================================== test session starts ======================================================================
platform linux -- Python 3.10.15, pytest-7.3.1, pluggy-1.5.0
rootdir: /home/fdalmau/git/wazuh/framework
configfile: pytest.ini
plugins: anyio-4.1.0, metadata-3.1.1, trio-0.8.0, tavern-1.23.5, cov-4.1.0, asyncio-0.18.1, html-2.1.1
asyncio: mode=auto
collected 10 items                                                                                                                                              

framework/wazuh/core/tests/test_wlogging.py ..........                                                                                                    [100%]

====================================================================== 10 passed in 0.05s =======================================================================
```

</details>

### CODE REVIEW

File: framework/wazuh/core/config/models/management_api.py [hunk 1]
```diff
@@ -91,8 +91,6 @@ class ManagementAPIConfig(WazuhConfigBaseModel):
         Configuration for management API intervals. Default is an instance of ManagementAPIIntervals.
     ssl : APISSLConfig
         SSL configuration for the management API. Default is an instance of APISSLConfig.
-    logging : RotatedLoggingConfig
-        Logging configuration for the management API. Default is an instance of LoggingWithRotationConfig.
```
Comment (GGP1) [inline framework/wazuh/core/config/models/management_api.py:95]: The field still exists but its docstring was removed, I think we should restore it?

File: framework/wazuh/core/tests/test_wlogging.py [hunk 2]
```diff
@@ -19,181 +19,42 @@
...
     """Test if WazuhLogger __init__ method initialize all attributes properly."""
-    # To bypass the checking of the existence of a valid Wazuh install
-    with patch('os.path.join'):
-        wlogging.WazuhLogger(foreground_mode=mock_foreground_mode, log_path=mock_log_path, tag=mock_tag,
-                             debug_level=mock_debug_level, logger_name=mock_logger_name,
-                             custom_formatter=mock_formatter, max_size=mock_max_size)
-    for x in [mock_formatter, mock_logger_name, mock_debug_level, mock_foreground_mode,
-              mock_logger]:
+
+    wlogging.WazuhLogger(tag=mock_tag,debug_level=mock_debug_level,
```
Comment (GGP1) [inline framework/wazuh/core/tests/test_wlogging.py:44]:
```suggestion
    wlogging.WazuhLogger(tag=mock_tag, debug_level=mock_debug_level,
```

**Review by GGP1 — CHANGES_REQUESTED** [review]

**Review by GGP1 — APPROVED** [review]
LGTM. Just left two non-blocking requested changes.

File: api/api/alogging.py [hunk 3]
```diff
@@ -2,15 +2,13 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
 
-import collections
 import logging
 import json
 import re
-from pythonjsonlogger import jsonlogger
```
Comment (nico-stefani) [inline api/api/alogging.py:9]: We need to remove this from the requirements https://github.com/wazuh/wazuh/blob/1f351bba9968c5a0557ea9fddea159ce26b3471d/framework/requirements.txt#L22

**Review by nico-stefani — CHANGES_REQUESTED** [review]

File: api/api/alogging.py (hunk 3 above)
Comment (fdalmaup) [reply api/api/alogging.py:9]: You are right, the embedded interpreter downloads the dependencies at installation time.

**Review by nico-stefani — APPROVED** [review]
LGTM

**Pull request merged**